external/vcm/vcm/catalog.yaml

```diff
@@ -998,3 +998,27 @@ sources:
         access: read_only
       urlpath: "gs://vcm-ml-code-testing-data/c8-grids-regression-testing/landseamask.zarr/"
       consolidated: true
+
+  grid/ne30:
+    description: Lat, lon, and area of NE30 data
+    driver: zarr
+    metadata:
+      grid: ne30
+      variables:
+        - area
+        - lat
+        - lon
+    args:
+      urlpath: "gs://vcm-scream/latLonArea/ne30/ne30.zarr/"
+      consolidated: true
+
+  landseamask/ne30:
+    description: land_sea_mask of NE30 data
+    driver: zarr
+    metadata:
+      grid: ne30
+      variables:
+        - land_sea_mask
```

Comment:
I don't *think* it will cause issues, but just FYI the FV3 land sea mask includes a sea-ice mask (value=2), whereas it seems the SCREAM one is just 0 (ocean) or land (1).

Reply:
Right, we only have 0 and 1 in this land sea mask. I'm waiting to swap this to scream's land sea mask when I have access to it, not sure if that one will have sea ice.